test/HTS.t.sol

```diff
@@ -218,6 +218,9 @@ contract HTSTest is Test, TestSetup {
         bytes[] memory metadata = new bytes[](0);
         int64 initialTotalSupply = 10000000005000000;
         IHederaTokenService.TokenInfo memory tokenInfo;
+        IHederaTokenService.TokenKey[] memory keys = new IHederaTokenService.TokenKey[](1);
+        keys[0].keyType = 0x10; // Supply key
+        keys[0].key.contractId = msg.sender;
```

Comment:
let's set some ECDSA key to test this instead, so if we change the `contractId` semantics this test still passes.